stdlib/string.gr

```diff
@@ -930,7 +1021,8 @@ export let encodeAtWithBom = (s, encoding, dest, destPos) => {
 // @param includeBom: Bool - Whether to include the byte-order marker in the encoded output
 // @returns Bytes
 @disableGC
-let encodeHelp = (s: String, encoding: Encoding, includeBom: Bool) => {
+let rec encodeHelp = (s: String, encoding: Encoding, includeBom: Bool) => {
+  Memory.incRef(WasmI32.fromGrain((+)))
```

Comment:
I asked this on another chunk of code, but why is `(+)` incref'd but never decref'd in this function?

Reply:
We invoke `+` on the next line, which follows the normal calling convention (meaning that it will `decRef` itself, so it should be `incRef`ed before the call)